admin-dev/themes/new-theme/js/components/translatable-input.js

```diff
@@ -40,11 +40,7 @@ class TranslatableInput {
     this.localeButtonSelector = opts.localeButtonSelector || '.js-locale-btn';
     this.localeInputSelector = opts.localeInputSelector || '.js-locale-input';
 
-    $('body').on(
-      'click',
-      this.localeItemSelector,
-      this.toggleLanguage.bind(this),
-    );
+    $('body').on('click', this.localeItemSelector, this.toggleLanguage.bind(this));
```

Comment:
No reason to change the indentation :thinking:

Reply:
I think it's an autofix from his IDE, as there are some ESLint errors

Reply:
Yeah I think it is, will fix it in next commit.